pom.xml

```diff
@@ -777,6 +792,21 @@
                             <ignoredUnusedDeclaredDependencies>
                                 <ignoredUnusedDeclaredDependency>*</ignoredUnusedDeclaredDependency>
                             </ignoredUnusedDeclaredDependencies>
+                            <ignoredNonTestScopedDependencies>
+                                <ignoredNonTestScopedDependency>org.eclipse.jetty:jetty-servlet</ignoredNonTestScopedDependency>
+                                <ignoredNonTestScopedDependency>org.apache.knox:gateway-util-urltemplate</ignoredNonTestScopedDependency>
+                                <ignoredNonTestScopedDependency>org.jboss.shrinkwrap.descriptors:shrinkwrap-descriptors-api-base</ignoredNonTestScopedDependency>
+                                <ignoredNonTestScopedDependency>org.apache.knox:gateway-spi</ignoredNonTestScopedDependency>
+                                <ignoredNonTestScopedDependency>com.nimbusds:nimbus-jose-jwt</ignoredNonTestScopedDependency>
+                                <ignoredNonTestScopedDependency>org.apache.httpcomponents:httpcore</ignoredNonTestScopedDependency>
+                                <ignoredNonTestScopedDependency>org.apache.knox:gateway-server</ignoredNonTestScopedDependency>
+                            </ignoredNonTestScopedDependencies>
+                            <ignoredUsedUndeclaredDependencies>
+                                <ignoredUsedUndeclaredDependency>org.glassfish.hk2.external:jakarta.inject</ignoredUsedUndeclaredDependency>
+                                <ignoredUsedUndeclaredDependency>jakarta.ws.rs:jakarta.ws.rs-api</ignoredUsedUndeclaredDependency>
+                                <ignoredUsedUndeclaredDependency>jakarta.annotation:jakarta.annotation-api</ignoredUsedUndeclaredDependency>
+                            </ignoredUsedUndeclaredDependencies>
+                            <skip>${maven.dependency.skip}</skip>
```

Comment:
What's the plan with all these ignores?
Shall we try to eliminate them once this is merged?